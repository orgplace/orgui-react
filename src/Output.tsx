import classNames from 'classnames';
import React from 'react';

const OuiOutput: React.FC<React.OutputHTMLAttributes<HTMLOutputElement>> = (props) => {
  return (
    <output {...props} className={classNames(
      'ouiOutput',
      props.className,
    )}>{props.children}</output>
  );
}

export { OuiOutput };
