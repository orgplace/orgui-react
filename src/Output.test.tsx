import React from 'react'
import { OuiOutput } from './Output'
import renderer from 'react-test-renderer';

test('OuiOutput', function() {
	const component = renderer.create(<OuiOutput>test</OuiOutput>);

	let tree = component.toJSON();
	expect(tree).toMatchSnapshot();
});
