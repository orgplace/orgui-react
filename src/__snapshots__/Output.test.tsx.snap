// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OuiOutput 1`] = `
<output
  className="ouiOutput"
>
  test
</output>
`;
